cmd/internal/cli/config_global_linux.go

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) 2019, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package cli
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/sylabs/singularity/docs"
+	"github.com/sylabs/singularity/internal/app/singularity"
+	"github.com/sylabs/singularity/internal/pkg/sylog"
+	"github.com/sylabs/singularity/pkg/cmdline"
+)
+
+// -s|--set
+var globalConfigSet bool
+var globalConfigSetFlag = cmdline.Flag{
+	ID:           "globalConfigSetFlag",
+	Value:        &globalConfigSet,
+	DefaultValue: false,
+	Name:         "set",
+	ShortHand:    "s",
+	Usage:        "set value of the configuration directive (for multi-value directives, it will add it)",
+}
+
+// -u|--unset
+var globalConfigUnset bool
+var globalConfigUnsetFlag = cmdline.Flag{
+	ID:           "globalConfigUnsetFlag",
+	Value:        &globalConfigUnset,
+	DefaultValue: false,
+	Name:         "unset",
+	ShortHand:    "u",
+	Usage:        "unset value of the configuration directive (for multi-value directives, it will remove matching values)",
+}
+
+// -g|--get
+var globalConfigGet bool
+var globalConfigGetFlag = cmdline.Flag{
+	ID:           "globalConfigGetFlag",
+	Value:        &globalConfigGet,
+	DefaultValue: false,
+	Name:         "get",
+	ShortHand:    "g",
+	Usage:        "get value of the configuration directive",
+}
+
+// -r|--reset
+var globalConfigReset bool
+var globalConfigResetFlag = cmdline.Flag{
+	ID:           "globalConfigResetFlag",
+	Value:        &globalConfigReset,
+	DefaultValue: false,
+	Name:         "reset",
+	ShortHand:    "r",
+	Usage:        "reset the configuration directive value to its default value",
+}
+
+// -d|--dry
```

Comment:
Please use `--dry-run` for consistency, as we already have that for `cache clean`

https://github.com/sylabs/singularity/blob/3fb66cde8cc5f0d077a447cdf16b643ecf3a6e53/cmd/internal/cli/cache_clean_linux.go#L60